backend/locales/en-US.json

```diff
@@ -264,6 +264,15 @@
             "updating": "Updating"
         }
     },
+    "WarnThirdParty":{
+        "title_zip": "Third party plugin install",
+        "title_repo": "Third party store selection",
+        "button_processing_one": "Please wait {{timer}} second",
+        "button_processing_many": "Please wait {{timer}} seconds",
+        "button_idle": "I accept the risks",
```

Comment:
This is a bit verbose, I don't think we need to outright say you "accept the risks" as it makes the button quite large. Something like "Continue" or "Accept" should suffice.